workshop.wwwapi/Models/AppointmentDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace workshop.wwwapi.Models
+{
+    public class AppointmentDto
```

Comment:
DTOs are better placed in their own DTO folder so that they are not part of the data model, which represents the database data.